src/components/Header/Header.js

```diff
@@ -11,18 +11,22 @@ import styles from './Header.module.scss';
 
 import { ReactSVG } from 'react-svg';
 
-const Header = ({ headerIsWhite }) => {
+const Header = ({ headerIsWhite, forwardedRef, backgroundGradient }) => {
   const { logotype } = useHeaderAssets();
   const { isOpened, handleTogglingIsOpened } = useContext(MenuContext);
 
   const menuStatus = isOpened ? 'opened' : 'closed';
-
   const barStyles = classNames(styles.bar, 'pageWrapper');
   const headerStyles = classNames(styles.container, {
     [styles.isWhite]: headerIsWhite && !isOpened,
   });
+
+  const setBackgroundGradient = classNames(headerStyles, {
+    [styles.gradientIsRemoved]: backgroundGradient,
```

Comment:
naming looks confusing

why we're adding class `gradientIsRemoved` when `backgroundGradient` is `true`? Should we remove gradient when `backgroundGradient` prop is `false`?